src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,60 @@
+package duke.commands;
+
+import duke.*;
+import duke.util.OutputUi;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.tasks.Task;
```

Comment:
Hi Tiffany, I noticed that some imports are not arranged in lexicographical order. For example, 'duke.tasks.Task' import could be placed before 'duke.util' packages. This would make the ordering of import statements more consistent.

Reply:
I also noticed that the .class files are in your main folder. If I am not wrong, intellij should auto place the files into the build folder? Perhaps you could take a look into that. Cheers 👍